### ISSUE TITLE
Test for WCAG failure F63: Providing link context only in content that is not related to the link

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#159]
This describes a failure condition when the context needed for understanding the purpose of a link is located in content that is not programmatically determined link context. If the context for the link is not in the same sentence, paragraph, list item, or table cell as the link, then the user will not be able to find out where the link is going with any ease. If the user must leave the link to search for the context, the context is not programmatically determined link context and this failure condition occurs.
### Failure Example 1: A News Service

A news service lists the first few sentences of an article in a paragraph. The next paragraph contains the link "Read More...". Because the link is not in the same paragraph as the lead sentence, the user cannot easily discover what the link will let him read more about. 
### Failure Example 2: Downloading a Free Player

An audio site provides links to where its player can be downloaded. The information about what would be downloaded by the link is in the preceding row of the layout table, which is not programmatically determined context for the link.
### Failure Example 3: Using a Definitions List

In HTML and XHTML, definition lists provide a programmatic association between the term and its definition. So theoretically, a link provided in a definition could use the definition term as its context. However, support is particularly bad for definitions lists, and there would be no way for users of today's assistive technology to discover the context using a definition list alone. Definition lists are a useful mechanism for providing associative relationships, but at this moment in time could not be considered sufficient for Success Criterion 2.4.2.

Testprocedure
1. Locate content needed to understand how link text describes the purpose of the link.
2. Check whether the content is contained in the same sentence, paragraph, list item, or table cell, or in the preceding heading.
